doc/man1/x509.pod

```diff
@@ -578,7 +578,8 @@ Dump any field whose OID is not recognised by OpenSSL.
 B<sep_multiline>
 
 These options determine the field separators. The first character is
-between RDNs and the second between multiple AVAs (multiple AVAs are
+between Relative Distinguished Names (RDNs) and the second is between 
+multiple Attribute Value Assertions (AVAs, multiple AVAs are
```

Comment:
I suggest `(AVAs). Multiple ` Basically, don't hide the second sentence in parentheses.